libs/designsystem/src/lib/components/page-local-navigation/page-local-navigation-item.ts

```diff
@@ -0,0 +1,26 @@
+import { ThemeColor } from '@kirbydesign/core';
+
+export type LocalNavigationItemBadge = {
+  icon?: string;
+  content?: string;
+  description?: string;
+  isCustomIcon?: boolean;
+  themeColor: ThemeColor;
+};
```

Comment:
Loose/fluffy typing - consider something along the lines:
type BadgeIcon = {
  name: string;
  isCustom: boolean;
};
type BadgeText = {
  text: string;
};

type TabBadge = {
  content: BadgeIcon | BadgeText;
  description: string;
  color?: ThemeColor;
};

export interface NavigationTab {
  text: string;
  badge?: TabBadge;
}